### PR TITLE
Created a seaborn.md file in /python/concepts/seaborn

### PR DESCRIPTION
**Pull request by BoboyeOkeya**
### Description

I have created a new folder named seaborn under /python/concepts/ and I created a seaborn.md file under this seaborn folder. I made these changes since I selected the issues titled: [New Topic] Python:Seaborn #874 because I was asked to contribute to the CodeAcademy Docs as part of my learning in the CodeAcademy's Learn Git/GitHub Course.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/seaborn/seaborn.md
Comment (SSwiniarski) [inline content/python/concepts/seaborn/seaborn.md:10]:
```suggestion
[Seaborn](https://seaborn.pydata.org/) is a popular open-source Python library, used for visualizing the explorative statistical plots of data. It is a basic libary used to extract various information that can provide an understanding of a dataset. To use Seaborn's objects and methods, it will need to be installed first. There are instructions on how to do so on the [Seaborn website](https://seaborn.pydata.org/installing.html).
```

File: content/python/concepts/seaborn/seaborn.md
Comment (SSwiniarski) [inline content/python/concepts/seaborn/seaborn.md:1]: Because this is a topic, it needs to move to content/seaborn/seaborn.md

File: content/python/concepts/seaborn/seaborn.md
Comment (SSwiniarski) [inline content/python/concepts/seaborn/seaborn.md:12]:
```suggestion
Seaborn provides a wide variety of objects and methods for use in accessing, analyzing, and visualizing data. A few of the more significant ones are listed below.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
I did a first review and made a few comments.

**Comment by SSwiniarski** [comment]
In addition to moving this file to content/seaborn/seaborn.md, according to the issue you're working on, there should be one or two concept entries to go with this entry which would live under content/seaborn/concepts.

**Comment by BoboyeOkeya** [comment]
> In addition to moving this file to content/seaborn/seaborn.md, according to the issue you're working on, there should be one or two concept entries to go with this entry which would live under content/seaborn/concepts.

**Comment by BoboyeOkeya** [comment]
> In addition to moving this file to content/seaborn/seaborn.md, according to the issue you're working on, there should be one or two concept entries to go with this entry which would live under content/seaborn/concepts.

Hello @SSwiniarski  I am so sorry. I am lost at the moment. Based on the last commit I made from my branch "
[Edited line 12 of Seaborn.md by deleting 'manipulating'] ". Are you saying that I should create one or more concepts? 

I thought I would get notified about this new review of yours, but I did not. so it was a bit confusing sorry

**Comment by SSwiniarski** [comment]
> > In addition to moving this file to content/seaborn/seaborn.md, according to the issue you're working on, there should be one or two concept entries to go with this entry which would live under content/seaborn/concepts.
> 
> Hello @SSwiniarski I am so sorry. I am lost at the moment. Based on the last commit I made from my branch " [Edited line 12 of Seaborn.md by deleting 'manipulating'] ". Are you saying that I should create one or more concepts?
> 
> I thought I would get notified about this new review of yours, but I did not. so it was a bit confusing sorry

@BoboyeOkeya, it's not based on your commit. It's based on the issue you're working on. In addition to the topic file, it wants one or two concept files to go underneath it.

**Comment by BoboyeOkeya** [comment]
Hello @SSwiniarski  do you think the format would be similar to this example https://www.codecademy.com/resources/docs/matplotlib/pyplot

**Comment by SSwiniarski** [comment]
@BoboyeOkeya, It depends on what it is you're documenting. What entry would you be planning to make?

**Comment by BoboyeOkeya** [comment]
hi @SSwiniarski I am working on the load_dataset. Here is an overview of the raw file
image

**Comment by BoboyeOkeya** [comment]
Hello @SSwiniarski please can I get some help.
1. I cant push from the local branch (boboye_newtopic_seaborn) into the origin 
2. I can't checkout to my main branch anymore, because I reset the head of my boboye_newtopic_seaborn branch.
I would really appreciate a guide, thanks.
Here is a picture of the error I keep getting in both cases
image

**Comment by BoboyeOkeya** [comment]
I think I would have to:
1. Delete the cloned repository on my machine. 
2. Reclone it again from my forked repository. 
3. Copy the seaborn folder into it again. Since it contains seaborn.md and the new concepts I just wrote.

I hope this would not create any problems @SSwiniarski ?

**Comment by SSwiniarski** [comment]
@Dusch4593, can you help @BoboyeOkeya?

**Comment by Dusch4593** [comment]
Hey @BoboyeOkeya, I can try to help! 😄

First, I'm just curious, what do you get in your console when you run `git status`?

**Comment by BoboyeOkeya** [comment]
Hello @Dusch4593 this is what i did so far to fix the issue:
 1. I forced checkout from boboye_newtopic_seaborn to main. 
 2. Then I remerged main into boboye_newtopic_seaborn.
 3. Then I added all files again and made a new commit.
But I still cant push from boboye_newtopic_seaborn into origin.

Here is what I get when I run git status:
image

**Comment by Dusch4593** [comment]
@BoboyeOkeya Are you getting the same error when you try to push to origin or a different one?

**Comment by BoboyeOkeya** [comment]
@Dusch4593 Yes I am still getting the same error. Here is the output of the push command
image

**Comment by Dusch4593** [comment]
@BoboyeOkeya Gotcha. According to [GitHub's documentation](https://docs.github.com/en/get-started/using-git/dealing-with-non-fast-forward-errors), for non-fast-forward errors like these, running a fetch and merge should reconcile this error. Have we tried running `git pull origin boboye_newtopic_seaborn`?

If we run this command, we will probably be taken to a new screen with a bunch of hashtags (#). Typing `wq` followed by "Enter" will save the changes and exit this screen.

After this, you should now be able to successfully run `git push origin boboye_newtopic_seaborn`. 🤞🏻🤞🏻

**Comment by Dusch4593** [comment]
@BoboyeOkeya Let me know if you run into any problems, though. ☺️

**Comment by BoboyeOkeya** [comment]
@Dusch4593 It worked. Thank you so much. I will keep this tip in mind for future references.

@SSwiniarski Please I hope the new pull request won't cause any conflicts, because it includes the seaborn.md file that I uploaded initially. And it also includes new files such as built-in-functions.md and load-dataset.md

**Comment by BoboyeOkeya** [comment]
hello again @Dusch4593 it seems my work is failing some of the tests. Please how do I fix this or should I just wait for review from @SSwiniarski ??? 
Thanks so much

**Comment by BoboyeOkeya** [comment]
@Dusch4593 Okay, thanks again :)

File: content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md
Comment (SSwiniarski) [inline content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md:35]:
```suggestion
|        `cache`       |            boolean         | The default value is `True`or `1` and it allows the function to initially load the dataset from a local cache directory, after which it saves to this cache directory if a download is needed.  |
```

File: content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md
Comment (SSwiniarski) [inline content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md:36]:
```suggestion
|     `data_home`      |            str             | Specifies the local directory where the cache data is stored. To view this directory, use the function `print(seaborn.get_data_home())`.        |   
```

File: content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md
Comment (SSwiniarski) [inline content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md:37]:
```suggestion
|       `kws`          |   dict(keys & values)      | Optional and extra keyword arguments that are passed to the Pandas function `.read_csv`. This Pandas function reads a csv file and returns a DataFrame. |
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@BoboyeOkeya, just a few more comments.

File: content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md
Comment (BoboyeOkeya) [reply content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md]: Done :)

File: content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md
Comment (BoboyeOkeya) [reply content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md]: Done :)

File: content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md
Comment (BoboyeOkeya) [reply content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md]: Done in a single push update :)

**Review by SSwiniarski — APPROVED** [review]
@BoboyeOkeya Everything looks good for a second review.

**Comment by BoboyeOkeya** [comment]
Thanks @SSwiniarski , it seems my submissions are still failing some of the tests. Do I wait for a third review? ;)

**Comment by SSwiniarski** [comment]
@BoboyeOkeya, one of the maintainers will be by for the second review.  As far as the failing tests, look at the documentation [here](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md), under "I submitted a PR but tests are failing, how do I fix them?" If you still have problems after reviewing the instructions there, one of the maintainers will be able to assist you.

**Comment by BoboyeOkeya** [comment]
@SSwiniarski Thank you for the feedback. I am not familiar with how to verify the tests and all. But I have installed npm and yarn and I will look into it. Do I still wait for one of the maintainers? Thanks

**Comment by SSwiniarski** [comment]
@BoboyeOkeya one of the maintainers will do a second review, but you don't have to wait to do npm/yarn etc.

**Comment by BoboyeOkeya** [comment]
Okay thank you @SSwiniarski !

File: content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md
Comment (Dusch4593) [inline content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md:25]:
```suggestion
seaborn.load_dataset(name, cache=True, data_home=None, **kws)
```

File: content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md
Comment (Dusch4593) [inline content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md:37]:
```suggestion
|       `name`         |            `str`             | Name of the dataset (in CSV format) from examples repository. |
|        `cache` (optional)      |            `boolean`         | Initially loads the dataset from a local cache directory, after which it saves to this cache directory if a download is needed. |
|     `data_home`   (optional)   |            `str`             | Specifies the local directory where the cache data is stored; can be viewed with `seaborn.get_data_home()`. |
|       `kws`   (optional)       |   `dict` (keys & values)      | Extra keyword arguments that are passed to the [pandas `.read_csv()` function](https://www.codecademy.com/resources/docs/pandas/built-in-functions/read-csv). |
```

Applying these edits will most certainly mess up the spacing of this table, but don't worry about it. 😄

Before we merge, I will go in and format/lint the file(s).

File: content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md
Comment (Dusch4593) [inline content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md:40]:
```suggestion

The following example accesses, loads, and prints the contents of the `flights.csv` file via the `.load_dataset()` function:

```

Comment (Dusch4593) [reply content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md:40]: For Docs entries, let's precede an example code snippet with 1-2 sentences describing what's about to happen in the example.

File: content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md
Comment (Dusch4593) [inline content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md:46]:
```suggestion
import seaborn as sns

data = sns.load_dataset("flights")
print(data)
```

No need for this comment with suggested text above the snippet.

File: content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md
Comment (Dusch4593) [inline content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md:49]:
```suggestion
This will output the following:
```

**Comment by BoboyeOkeya** [comment]
Hello @Dusch4593 , I am on it. Thanks

File: content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md
Comment (BoboyeOkeya) [reply content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md:40]: Nice, will keep that in mind

File: content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md
Comment (BoboyeOkeya) [reply content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md:37]: I am a bit new to this format/lint. But I am aware that they are mean for automated testing of one's work. I had node.js and also installed the packages such as yarn, prettier, markdownlint, and jest. But I have no idea how all these work, so please I would appreciate if you can help with some materials on how to learn about these and software testing xD

**Review by Dusch4593 — APPROVED** [review]
@BoboyeOkeya This looks great! Thank you for making those changes 😄

I made a few more here in f63390d581f16bacbc82685fdb3ccfa9edfd09ed, including lint/formatting. Hopefully, all the tests should now be passing 🚀

**Comment by Dusch4593** [comment]
One other thing to note. I edited the `CatalogContent` of the files and opted for the slug for [this Codecademy course path for Data Science](https://www.codecademy.com/learn/paths/data-science-foundations) since it seems most relevant to this content.

cc: @BoboyeOkeya @SSwiniarski

**Comment by Dusch4593** [comment]
Congrats and thank you for contributing to Codecademy Docs, @BoboyeOkeya! 🚀 🎊 🎉

The entries should now be live.

Seaborn (topic entry): https://www.codecademy.com/resources/docs/seaborn
Built-in Functions: https://www.codecademy.com/resources/docs/seaborn/built-in-functions
`.load_dataset()`: https://www.codecademy.com/resources/docs/seaborn/built-in-functions/load-dataset

Currently, your Codecademy profile is anonymous on the entry page. To make it visible, you'll need to add your GitHub username to your Codecademy profile, log out of Codecademy, and log back in through your GitHub account.

Screen Shot 2022-11-28 at 6 13 59 PM

Looking forward to your next Docs contribution! 😄

**Comment by BoboyeOkeya** [comment]
Thanks for the support and good work to you too 🎉🎉🎉 @Dusch4593 !!!. I have tried to make my profile visible by doing as mentioned, but I can't seem to see a difference though.

**Comment by BoboyeOkeya** [comment]
Hello 🥰 🥰 🥰 @SSwiniarski  and @Dusch4593, thank you so much again for the collaboration. Please may I ask on how to get my name on the contributor list as shown here ?
image
I have adjusted my codecademy details as instructed but it still does not show up
image

**Comment by Dusch4593** [comment]
Hey @BoboyeOkeya! Looks like you did everything right and your profile seems to be showing up. 😄

Screen Shot 2022-11-30 at 12 20 45 PM

Sometimes an edit to an existing entry page takes a little time. 😅

**Pull request merged**
